sources/packages/backend/apps/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -91,8 +91,8 @@ export class GetApplicationDataDto extends GetApplicationBaseDTO {
   applicationAssessmentStatus?: AssessmentStatus;
   applicationPIRDeniedReason?: string;
   applicationCOEDeniedReason?: string;
-  programYearStartDate: Date;
-  programYearEndDate: Date;
+  programYearStartDate: string;
```

Comment:
I added this file to the tech debt: https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1495